src/KubernetesClient/Authentication/OidcTokenProvider.cs

```diff
@@ -14,23 +16,33 @@ public class OidcTokenProvider : ITokenProvider
         private string _idToken;
         private string _refreshToken;
         private string _accessToken;
-        private DateTime _expiry;
+        private DateTimeOffset _expiry;
```

Comment:
any other concern why change this to datetimeoffset? just wondering. 

i do see
```
bumps IdentityModel.OidcClient
this causes the change from DateTime to DateTimeOffset
``

but did not understand why

Reply:
it looks like this was changed to resolve an issue with cross timezone information
- https://github.com/IdentityModel/IdentityModel.OidcClient/issues/273
- https://github.com/IdentityModel/IdentityModel.OidcClient/pull/274